README.md

```diff
@@ -5,6 +5,29 @@ The Operator Courier is used to build, validate and push Operator Artifacts.
 It is currently in a pre-alpha stage.
 
 ## Usage
+
+### Command Line Interface
+To install the operator courier project to use from the command line, just install the latest release from [PyPI](https://pypi.org/project/operator-courier/):
+
+```bash
+$ pip3 install operator-courier
+$ operator-courier
+```
+
+Once the project is installed, to generate a bundle and push it to a quay.io app registry just use `operator-courier push`
```

Comment:
`generate a bundle` seems to imply the tool is generating the files. What is required here is a note about how the user should organize the files.

Reply:
@aravindhp courier does generate the bundle yaml. I can add more specifics here though to define exactly what is expected and what is generated.

Reply:
Are we re-defining what a [bundle](https://github.com/operator-framework/operator-registry#manifest-format) is? But yes, calling out what is expected will help.

Reply:
@aravindhp Here we are using the word `bundle` to describe the yaml blob that is pushed up to quay (similarly to how it is used in the community-operators repo). If and when we update courier to use the nested bundles, we should update our readme and source to use the correct terms -- AFAIK there is no correct term right now for the yaml blob using the new nested format.

Reply:
updated.